src/constants/upload_types.js

```diff
@@ -0,0 +1,6 @@
+const UPLOAD_TYPES = {
```

Comment:
Consider opening an issue to refactor all self to pod interactions as 'self' including 'self-fetch' for file searching and 'cross' for 'cross-fetch'.

Reply:
Created Issue #161 for this.